Android/app/src/main/java/app/intra/net/go/GoVpnAdapter.java

```diff
@@ -87,7 +88,7 @@ String make(String template) {
   private ParcelFileDescriptor tunFd;
 
   // The Intra session object from go-tun2socks.  Initially null.
-  private intra.Tunnel tunnel;
+  private Session tunnel;
```

Comment:
If you are renaming the type, you should rename the variable too
```suggestion
  private Session session;
```

Reply:
renamed